libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -112,12 +116,35 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("_H_OFFSET", 11, AP_OSD, h_offset, 32),
 
+#if OSD_CRSF_PANELS_ENABLED
+    // @Param: _W_RSSI
+    // @DisplayName: RSSI warn level (in %)
+    // @Description: Set level at which RSSI item will flash (-dBm)
+    // @Range: -128 0
+    // @User: Standard
+    AP_GROUPINFO("_W_RSSI", 12, AP_OSD, warn_rssi, -100),
+
+    // @Param: _W_LQ
+    // @DisplayName: CRSF LQ warn level (in %)
+    // @Description: Set level at which CRSF_LQ item will flash (%)
+    // @Range: 0 100
+    // @User: Standard
+    AP_GROUPINFO("_W_LQ", 33, AP_OSD, warn_lq, 50),
```

Comment:
ditto name

Reply:
I haven't changed this to keep consistency. All other warning level parameters follow this "_W_" pattern, so I chose to keep it that way.